### PR TITLE
[WIP][DO_NOT_MERGE] Console/cr/update

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Updates for the console CR.
/assign @spadgett 
Related to [global config](https://github.com/openshift/api/pull/159).

### CODE REVIEW

File: manifests/00-crd.yaml [hunk 1]
```diff
@@ -1,13 +1,17 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: consoles.console.openshift.io
+  name: consoleoperatorconfigs.console.operator.openshift.io
```
Comment (benjaminapetersen) [inline manifests/00-crd.yaml]: @spadgett fyi updating the CRD to conform with the most current patterns.

File: manifests/00-crd.yaml [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
-    plural: consoles
-    singular: console
-  scope: Namespaced
-  version: v1alpha1
+    kind: ConsoleOperatorConfig
+    listKind: ConsoleOperatorConfigList
+    plural: consoleoperatorconfigs
+    singular: consoleoperatorconfig
+    categories:
+    - coreoperators
```
Comment (benjaminapetersen) [inline manifests/00-crd.yaml]: `oc get coreoperators -o yaml` should include this, the console operator config.

File: manifests/00-crd.yaml (hunk 1 above)
Comment (benjaminapetersen) [reply manifests/00-crd.yaml]: `consoleoperatorconfigs` is what this represents now. `console.operator.openshift.io` is our group. This follows the pattern from [kube api server operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_10_kube-apiserver-operator_01_config.crd.yaml#L4).

**Comment by spadgett** [comment]
RBAC policy will need to be updated, too

**Comment by benjaminapetersen** [comment]
outdated.

**Pull request closed without merging**